include/revng/ADT/UpcastablePointer.h

```diff
@@ -52,6 +52,15 @@ concept UpcastablePointerLike = Dereferenceable<T> and Upcastable<pointee<T>>;
 template<typename T>
 concept NotUpcastablePointerLike = not UpcastablePointerLike<T>;
 
+/// This is an upcasting helper `UpcastablePointer` below takes advantage of
+/// heavily. When possible, prefer the member function instead.
```

Comment:
I think a comma is missing or something.

Reply:
Right, good catch:
```cpp
/// This is an upcasting helper, `UpcastablePointer` below takes advantage of
/// heavily. When possible, prefer the member function instead.
```

Reply:
I usually explicitly use a word "done" of "fixed" in that case!